contents/handbook/people/team-structure/people.md

```diff
@@ -13,48 +13,86 @@ hideAnchor: true
-|----------------------------------------------------------------------|--------------------------------------------|--------|---------|
-| People want to stay with PostHog                                     | % Turnover per quarter                     | 5%     | 3.5%    |
-| People want to be involved in making PostHog the best place to work  | % Response rate of Engagement survey       | 90%    | 96%     |
-| People recommend PostHog as a great place to work.                   | Average score in engagement survey         | 90%    | 94%     |
-| People feel informed about what’s happening at PostHog               | Average score in engagement survey         | 90%    | 91%     |
-| People understand PostHog's goals and know how to contribute.        | Average score in engagement survey         | 90%    | 89%     |
-| People feel like they develop professionally.                        | Average score in engagement survey         | 85%    | 88%     |
-| People feel like they are being paid competitively.                  | Average score in engagement survey         | 80%    | 78%     |
-| People feel a sense of belonging.                                    | Average score in engagement survey.        | 80%    | 82%     |
+### Q2 2022
 
+**Objective:** Find and retain our world class team.
 
+**Key results:**
+- We hire 9 roles on time
+- Engagement from team surveys stays above 80% of [Cultureamp industry benchmarks](https://www.cultureamp.com/science/insights/bay-area)
+- 15% of applicants who convert to first round interviews are from under-represented backgrounds
+- We stay within 5% of forecast cash burn
 
 ## Slack channels
 
 [#team-people-and-ops](https://posthog.slack.com/messages/team-people-and-ops) - internally public, default for ops & people discussions
 [#private-people-ops](https://posthog.slack.com/messages/private-people-ops) - internally confidential, for minority of issues, e.g. salaries, candidate offers
+
+## Tools
+
+An operations team is never complete without a robust toolkit to allow us to automate processes and multiply our impact. Here is a summary of some of the tools we've found most effective for getting the job done. You can read more about them [here](https://posthog.com/blog/startup-ops-toolkit).
+
+### Hiring
+
+**Contracts & Offer Letters**
+
+Do _not_ pay a lawyer to create contract templates for you. Orrick has a set of [free templates](https://www.orrick.com/en/Total-Access/Tool-Kit/Start-Up-Forms/Employment-and-Consultant) you can use.
+
+Consider using an e-signature platform like [HelloSign](https://hellosign.com/) to avoid emailing PDFs back and forth forever.
+
+**HR, Recruiting, and Payroll**
+
+[Gusto](https://www.gusto.com/) is the industry standard for managing and paying US-based employees.
+
+If you're hiring abroad, or hiring contractors, consider using [Deel](https://www.letsdeel.com/).
```

Comment:
To hire abroad, or for hiring contractors, we use [Deel](https://www.letsdeel.com/).